library/billing_sftp_service.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * Executes the background service for billing, which sends EDI claims
+ * directly to the x-12 partner (if enabled)
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Ken Chapple <ken@mi-squared.com>
+ * @copyright Copyright (c) 2021 Ken Chapple <ken@mi-squared.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+set_time_limit(0);
+
+$ignoreAuth = true;
+$fake_register_globals = false;
+$sanitize_all_escapes = true;
+
+require_once __DIR__.'/../interface/globals.php';
```

Comment:
actually, should not need any of the code above (since it is already done in the library/ajax/execute_background_services.php script)